spi/common/core-spi/src/main/java/org/eclipse/edc/spi/agent/ParticipantAgent.java

```diff
@@ -52,4 +64,12 @@ public Map<String, Object> getClaims() {
     public Map<String, String> getAttributes() {
         return attributes;
     }
+
+    /**
+     * Returns the verified identity of the participant or null if one is not available.
+     */
+    @Nullable
+    public String getIdentity() {
+        return attributes.get(PARTICIPANT_IDENTITY);
```

Comment:
I avoided that at the moment because it would ripple more changes, particularly in the test classes. We can look to change that later once we decide if we are going to back-port this.